scripts/code_style.py

```diff
@@ -33,6 +33,17 @@
 def print_err(*args):
     print("Error: ", *args, file=sys.stderr)
 
+def print_warn(*args):
+    print("Warn:", *args, file=sys.stderr)
+
+# Print the file names that will be skipped and the help message
+def print_skip(files_to_skip):
```

Comment:
Nit: This function prints first to `stdout` then the `print_warn()` prints to `stderr`. This should print either always to `stdout` or always to `stderr`, to avoid problems with the ordering.